docs/docs/10.8-pure-render-mixin.md

```diff
@@ -20,6 +20,19 @@ React.createClass({
   }
 });
 ```
+Example using ES6 class syntax:
+```js
```

Comment:
Can you add an extra newline before and after `Example using ES6 class syntax:`?  To be consistent with the previous `Example:`.  Some markdown parsers don't like having the code block immediately following text.